ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -892,15 +902,15 @@ def test020_create_new_order_with_test_units(self):
             testunit_name = row_with_headers['Test Unit']
             self.base_selenium.LOGGER.info(" + Test unit : {}".format(testunit_name))
             self.assertIn(testunit_name, test_units_list)
-            
-    # will continue with us
+
     def test021_create_existing_order_with_test_units(self):
         """
         New: Orders: Create an existing order with test units
 
         LIMS-3268
         """
-        self.base_selenium.LOGGER.info('Running test case to create an existing order with test units')
+
+        self.base_selenium.LOGGER.info('Running test case to create a new order with test units')
         test_units_list = []
```

Comment:
you can define  a new function in test_unit api which return 'Qualitative' test unit  like this :
`        test_units = self.get_all_testunits_json()`
`for testunit in test_units:
        if testunit['typeName']=='Quantitative' :
                  return testunit
`

Reply:
@0xIslamTaha I have a question this method do the followings

Reply:
@dianaeldemerdash The test case is testing "The behavior of creating a test order with a test unit and the behavior of the analysis", So anything else could be done in the level of API. Here is the steps:
- Get the required test unit using the API 
- Use it to create an order via UI 
- Assert behaviors via UI